docs/source/platform/schema-registry.md

```diff
@@ -177,3 +166,12 @@ const server = new ApolloServer({
   }
 });
 ```
+
+<h2 id="benefits">Tools that use the schema registry</h2>
+
+An up-to-date schema in Apollo's schema registry enables a number of workflow benefits through integrations with other tools. A registered schema combined with:
+
+- The [Apollo VS Code extension](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo) provides built-in linting on queries by validating against the schema in your registry, and annotates fields on your queries with performance indicators collected in Apollo's trace warehouse.
+- [Schema validation](./schema-validation.html) creates a diff between your local schema and the last schema uploaded to the registry, and validates this diff against live traffic seen on your endpoint to make sure you never accidentally deploy a breaking schema change to production.
```

Comment:
I still think we should be consistent about the word we use (upload, update, register, publish, etc.) and I vote for "publish" or "register" as words that imply the compound action of cloud storage, ownership, and history tracking.

Reply:
Agreed! I went for "push" everywhere since that's the label of our CLI command.